ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -120,14 +120,17 @@
               </div>
             </v-col>
             <v-col cols="2" class="pt-5"
-              ><v-btn
+              ><v-btn v-if="!isExistingSecuredParty(result.code)"
```

Comment:
Please keep the `>` with its tag unless its self closing (ie `/>`

Reply:
Just keep this in mind, if you do see this, to fix the formatting :)

Reply:
Makes sense.